index.html

```diff
@@ -0,0 +1,159 @@
+        <li class="skill">bootstrap</li>
+        <li class="skill">Ruby</li>
+       </ul>
+     <a class="see-project-1" href="">See Project</a>
+
+     <div class="card-2">
+       <h2 class="prof-art">Profesional Art Printing Data</h2>
+       <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+       <ul class="skils">
+        <li>html</li>
+        <li class="skil">bootstrap</li>
+        <li class="skil">Ruby</li>
+     </ul>
+     </div>
+     <a class="see-project-2" href="">See Project</a>
+
+
+     <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li>html</li>
+       <li class="skil">bootstrap</li>
+       <li class="skil">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2" href="">See Project</a>
+
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li>html</li>
+       <li class="skil">bootstrap</li>
+       <li class="skil">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li>html</li>
+       <li class="skil">bootstrap</li>
+       <li class="skil">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li>html</li>
+       <li class="skil">bootstrap</li>
+       <li class="skil">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li>html</li>
+       <li class="skil">bootstrap</li>
+       <li class="skil">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2" href="">See Project</a>
+    
+</section>
+
+<section class="section-3">
+   <div class="about-me">
+      <h1 class="h31">About me</h1>
+      <p class="p31">Hello I’m a software developer! I can help you build a product ,feature or 
```

Comment:
- [ ] Kindly remove the border around your `about me` header and adjust the padding of your `get my resume` button to match the original design

![image](https://user-images.githubusercontent.com/98453979/213498257-8bed2289-a497-4c8f-a818-88cace8d771a.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/213498539-76f94902-e0ee-4d37-a980-52cbd2d0fca3.png)